src/components/common/Tree/CustomTreeItem.js

```diff
@@ -51,7 +126,7 @@ const CustomTreeItem = ({ itemId, expandedItems = [], selectedId }) => {
       return LoadingTreeItem;
     }
     const filteredChildren = children?.filter(
-      (_child, idx) => !isHidden(childrenTags?.get(idx)),
+      (_child, idx) => !isHidden(childrenTags?.get(idx)) && _child.type === "folder",
```

Comment:
`_child` is a syntax to say we don't use `child`. You can rename to `child` now that you use the variable.
Factor out `folder` there's probably already a constant `ITEM_TYPES` or similar.

Reply:
Solved